namegen.py

```diff
@@ -0,0 +1,35 @@
+import random
+def main():
```

Comment:
[[PEP 8]](https://www.python.org/dev/peps/pep-0008/)

It's recommended have two line breaks around functions, so before the start of line 2.

Example:

```
import bcrypt


def foo():
  pass
```